NBISweden/sda-cli
# Feature/download dataset

**Related issue(s) and PR(s)**  
This PR closes #387.


**Description**
A new feature in the sda-cli sda-download is implemented in order to download all the files of the dataset (decrypted)
by adding the flag `--dataset`.

**How to test**
Run the setup script from the root folder by typing: `bash .github/integration/setup/setup.sh`. When everything is ready and the containers are running download all the files of the `https://doi.example/ty009.sfrrss/600.45asasga`dataset by running: `./sda-cli sda-download -config testing/s3cmd-download.conf -datasetID https://doi.example/ty009.sfrrss/600.45asasga -url http://localhost:8080 -outdir test-dataset --dataset`.
Then check the `test-dataset` folder and 3 files in different paths will be present.
